tools/packaging/static-build/coco-guest-components/Dockerfile

```diff
@@ -2,27 +2,40 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu:22.04
+FROM ubuntu:20.04
```

Comment:
Aaaah, this rootfs being based on Ubuntu 20.04 is a pain.  This is not related to this PR, but we'll need to bump it at some point.

Reply:
Yeah - that might have been the easier option in the end to just bump the rootfs!